src/utilities/truncateString/truncateString.ts

```diff
@@ -1,15 +1,9 @@
 /** This function shortens a string, adding an elipsis to the end if longer than provided length. */
```

Comment:
ellipsis is only added if provided by user. By default nothing is added, so comment is incorrect 😱 
Maybe:
/** Shorten a string to a specified length, option to add additional string to the end, for example ellipsis '...'  */